apps/opencs/view/render/pagedworldspacewidget.cpp

```diff
@@ -791,6 +790,41 @@ CSVRender::Cell* CSVRender::PagedWorldspaceWidget::getCell(const osg::Vec3d& poi
         return 0;
 }
 
+CSVRender::Cell* CSVRender::PagedWorldspaceWidget::getCell(const CSMWorld::CellCoordinates& coords) const
+{
+    std::map<CSMWorld::CellCoordinates, Cell*>::const_iterator searchResult = mCells.find(coords);
+    if (searchResult != mCells.end())
+        return searchResult->second;
+    else
+        return nullptr;
+}
+
+void CSVRender::PagedWorldspaceWidget::setCellAlteredHeight(const CSMWorld::CellCoordinates& coords, int inCellX, int inCellY, float height)
+{
+    std::map<CSMWorld::CellCoordinates, Cell*>::iterator searchResult = mCells.find(coords);
+    if (searchResult != mCells.end())
+        searchResult->second->setAlteredHeight(inCellX, inCellY, height);
+}
+
+float* CSVRender::PagedWorldspaceWidget::getCellAlteredHeight(const CSMWorld::CellCoordinates& coords, int inCellX, int inCellY)
+{
+    std::map<CSMWorld::CellCoordinates, Cell*>::iterator searchResult = mCells.find(coords);
+    if (searchResult != mCells.end())
+        return searchResult->second->getAlteredHeight(inCellX, inCellY);
+    return nullptr;
+}
+
+void CSVRender::PagedWorldspaceWidget::resetAllAlteredHeights()
+{
+    std::map<CSMWorld::CellCoordinates, Cell *>::iterator iter (mCells.begin());
+
+    while (iter!=mCells.end())
+    {
+        iter->second->resetAlteredHeights();
+        ++iter;
+    }
```

Comment:
Might be better for readability:
```c++
for (const auto& cell : mCells)
    cell.second->resetAlteredHeights();
```

Reply:
It's true, but pagedworldspacewidget.cpp uses iterator-method in other similar places. I think it's therefore better to have constantly the same method, rather than change it in one function.

Reply:
It's not like anyone's going to look at the modern-style for loop and be confused about what's going on, so I'd say making your new code readable is better than making it consistent with old code.